buildkite/src/Jobs/CompareSignatures.dhall

```diff
@@ -0,0 +1,41 @@
+let Prelude =  ../External/Prelude.dhall
+let S = ../Lib/SelectFiles.dhall
+let Cmd =  ../Lib/Cmds.dhall
+let Pipeline = ../Pipeline/Dsl.dhall
+let Command = ../Command/Base.dhall
+let OpamInit = ../Command/OpamInit.dhall
+let Docker = ../Command/Docker/Type.dhall
+let Size = ../Command/Size.dhall
+let JobSpec = ../Pipeline/JobSpec.dhall
+in
+
+Pipeline.build
+   Pipeline.Config::
+     { spec =
+         let opamDirtyWhen =
+           [ S.exactly "src/opam" "export"
+           , S.exactly "scripts/setup-opam" "sh"
+           , S.strictly (S.contains "Makefile")
+           , S.exactly "buildkite/src/Command/OpamInit" "dhall"
+           , S.exactly "buildkite/scripts/cache-through" "sh"
+           ]
```

Comment:
Can you add `opamDirtyWhen` to the `OpamInit` module so we don't have to copy and paste this expression? And then while you're there you can fix the usage in `ClientSdk.dhall` also!